src/components/modal/v2/styles/globals/_variables.scss

```
@@ -3,6 +3,7 @@ $sticky-header-height-tablet-desktop: 60px;
 $sticky-header-radius: 40px;
 
 $content-wrapper-max-width-tablet: 640px;
+$content-wrapper-max-width-tablet-checkout: 424px;
 $content-wrapper-max-width-desktop: 840px;
 $content-wrapper-max-width-api-iframe: 100vw;
 $content-wrapper-border-radius: 10px;
```

```suggestion
$content-wrapper-border-radius: 10px;
$content-wrapper-border-radius-checkout: 32px;
```
Let's add a new variable for the border radius for the new checkout modals.